docs/api/topics/firefox_os_addons.rst

```diff
@@ -346,6 +379,6 @@ Publishing
     :param int version_id: The add-on version id
 
     :status 202: successfully published.
-    :status 403: not allowed to access this object.
+    :status 403: not allowed to access this object or disabled add-on.
```

Comment:
This sentence is a bit unclear. Perhaps: "not allowed to access this object, or you are trying to change properties on a disabled addon, which [is not allowed]()." (link to [a label inserted here](https://github.com/mozilla/zamboni/pull/3356/files#diff-e1e81b1c2b8c89a397b0df2556da9950R216))